#89 Slide Collision Projection Patch Documentation (#90)
Inbibo/adonisfx_docs: docs/release_notes.md

# Release Notes

## Version 1.1.2

### Bug Fixes

> - Fixed a bug in the projection of the Slide Collision Constraints that when having Compute Collisions ON and Keep Orientation ON was applying an incorrect amount of correction past the collision threshold when the simulated mesh intersected the collider more than the collision threshold distance. *AdonisFX-1258*

Reviewer: ```suggestion
- Fixed a bug in the Slide Collision Constraints that was overcorrecting in case of intersection with the collider with a magnitude greater than the collision threshold. *AdonisFX-1258*
```

Author: Done. I agree to be less verbose

> ### Considerations

Reviewer: I would remove this consideration

Author: Done!
